packages/authentication/src/keys.ts

```diff
@@ -9,7 +9,7 @@
 export namespace BindingKeys {
   export namespace Authentication {
     export const STRATEGY = 'authentication.strategy';
-    export const PROVIDER = 'authentication.provider';
-    export const METADATA = 'authenticate';
+    export const AUTH_ACTION = 'authentication.actions.authenticate';
+    export const METADATA = 'authentication.operation-metadata';
```

Comment:
Why do we use .  in `actions.authenticate` and - in `operation-metadata` as separator?

What's wrong with `actions-authenticate`?    Isn't . the namespace separator?